ExampleMod/Content/Tiles/ExampleHerb.cs

```diff
@@ -96,11 +101,16 @@ public class ExampleHerb : ModTile
 		}
 
 		public override void SetSpriteEffects(int i, int j, ref SpriteEffects spriteEffects) {
-			if (i % 2 == 1) {
+			if (i % 2 == 0) {
 				spriteEffects = SpriteEffects.FlipHorizontally;
 			}
 		}
 
+		public override void SetDrawPositions(int i, int j, ref int width, ref int offsetY, ref int height, ref short tileFrameX, ref short tileFrameY) {
```

Comment:
This seems unnecessary, height is already 20 and offsetY is already -1 from StyleAlch

Reply:
StyleAlch is -1, offsetY is set to -2 here, because vanilla herbs seem to have an offset regarding the placement preview for some reason. I added a comment to clarify this